src/core/components.py

```diff
@@ -793,3 +817,138 @@ def get_base_component_dict(self):
     def get_components(self):
         yield from super().get_components()
         yield from self.components
+
+
+class Panel(Component):
+    """A panel which can contain components."""
+
+    __slots__ = [
+        "components",
+        "width",
+        "is_card",
```

Comment:
What is a card?

Reply:
Essentially a panel with a background and border set.

Reply:
Shouldn't that be up to the front end like it is with everything else?

Reply:
Yep we've now gotten rid of this and added a transparent style to essentially mean that the container shouldn't have a background or border.